src/components/Menu/MenuItem/MenuItemSubHeader/MenuItemSubHeader.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { MenuItemSubHeaderProps } from '../MenuItem.types';
+import { mergeClasses } from '../../../../shared/utilities';
+import { MenuSize } from '../../Menu.types';
+
+import styles from '../menuItem.module.scss';
+
+export const MenuItemSubHeader: FC<MenuItemSubHeaderProps> = ({
+    text,
+    size,
+}) => {
+    const subHeaderClasses: string = mergeClasses([
+        styles.menuItemSubHeader,
+        { 'body3 semibold': size === MenuSize.small },
```

Comment:
would prefer we course correct on number based sizing classes. we should call small "sm", large "lg", etc. the numbers aren't intuitive.

Reply:
Or single letter sizing ($space-s), as seem below in this PR, but we should establish a standard way of denoting size.

Reply:
MenuSize.small is "small", not using numbers here

Reply:
sorry, was referring to the body3. More of a comment for near future consideration than something I'm looking for action on in this PR